include/verilated_timing.h

```diff
@@ -56,6 +56,36 @@
 #endif
 // clang-format on
 
+//=============================================================================
+// VlFileLine stores a SystemVerilog source code location. Used in VlCoroutineHandle for debugging
+// purposes.
+
+class VlFileLine final {
+    // MEMBERS
+#ifdef VL_DEBUG
+    const char* m_filename = nullptr;
+    int m_lineno = 0;
```

Comment:
```suggestion
    const int m_lineno = 0;
```

Reply:
Unfortunately can't do this, as I overwrite `VlFileLine` in forks.

I could redo forks so that `init` sets the line number if you want.

Also, for consistency, `m_filename` should be `const char* const` then.

Reply:
Thats OK.